### ISSUE TITLE
Outlook add-in: Minimum release for event-based activation with InsightMessage

### ISSUE THREAD

**Issue by lago-dev** [OfficeDev/office-js#2019]
Hi,
we have implemented event-based activation for a OnNewMessageCompose event to autolaunch our Outlook add-in and display an InsightMessage. Everything works very nicely with release 16.0.14228.20204 (Current Channel). 

## Expected Behavior
According to the documentation we have expected that both OnNewMessageCompose and InsightMessage will work on 16.0.14026.20334 (Monthly Enterprise Channel) in the meantime. 

## Problem / Current Behavior
When downgrading to 16.0.14026.20334 (Monthly Enterprise Channel) event-based activation does not work anymore. Guess we have misunderstood the documentation and event-based activation with InsightMessage it is still not part of the 16.0.14026.20334 release?

## Questions
Since we are having a customer waiting on this feature - would you please let us know which is the minimum release for both features (event-based activation with InsightMessage)? Will it be included in the next update of Semi Annual Enterprise Channel?

## I am referring to the documentation below: 
https://docs.microsoft.com/en-us/office/dev/add-ins/outlook/autolaunch
https://docs.microsoft.com/de-de/javascript/api/outlook/office.notificationmessages?view=outlook-js-preview

## Your Environment
* Platform: PC desktop 
* Host: Outlook for Windows 
* Office version number: 16.0.14228.20204 (Current Channel) + 16.0.14026.20334 (Monthly Enterprise Channel)
* Operating System: Windows 10
* Browser (if using Office Online): Edge, Chrome

Thanks for your help and kind  regards, 
Martin

**Comment by exextoc** [comment]
Thanks for reporting this issue. We have addressed it and the minimum build for the Monthly Enterprise Channel for the Event-Based Add-ins feature is 16.0.14131.20000.
This build will be released on 08/10/2021.
If this still doesn't work after 08/10/2021, let us know so that we can look further into it.

For Semi-Annual Channel, we just forked for July 2021, and this fork will be released to the Semi-Annual Channel in January 2022. This release will contain the Event-Based Add-ins feature.
